tests/EpisodeIntegrationServiceTests/ReceiveDataTests/ReceiveDataTests.cs

```diff
@@ -846,5 +847,36 @@ public async Task ProcessHistoricalParticipantDataAsync_ShouldLogErrorAndIncreme
         Mock.Get(_mockEventGridPublisherClientFactory.Object("participant")).Verify(x => x.SendEventAsync(It.IsAny<EventGridEvent>()), Times.Exactly(2));
     }
 
+    [TestMethod]
+    [DataRow("", null, DisplayName = "ParseNullableDateTime_ShouldReturnNull_WhenGivenNullOrEmptyString")]
+    [DataRow("2024-02-26 14:30:00+01", "2024-02-26 13:30:00", DisplayName = "ParseNullableDateTime_ShouldParseCorrectly_WhenGivenTimezoneFormat")]
+    [DataRow("2024-02-26 14:30:00", "2024-02-26 14:30:00", DisplayName = "ParseNullableDateTime_ShouldParseCorrectly_WhenNoTimezoneProvided")]
```

Comment:
Nit-pick: add sad path test when a malformed string is passed:

```suggestion
    [DataRow("2024-02-26 14:30:00 BadFormat", null, DisplayName = "ParseNullableDateTime_ShouldReturnNullForBadInput")]
```